controllers/remove_pods.go

```diff
@@ -168,6 +182,55 @@ func confirmPodRemoval(r *FoundationDBClusterReconciler, context ctx.Context, cl
 	return true, nil
 }
 
+func includeInstance(r *FoundationDBClusterReconciler, context ctx.Context, cluster *fdbtypes.FoundationDBCluster, removedProcessGroups map[string]bool) error {
+	adminClient, err := r.AdminClientProvider(cluster, r)
+	if err != nil {
+		return err
+	}
+	defer adminClient.Close()
+
+	addresses := make([]string, 0)
+
+	hasStatusUpdate := false
+
+	processGroups := make([]*fdbtypes.ProcessGroupStatus, 0, len(cluster.Status.ProcessGroups))
+	for _, processGroup := range cluster.Status.ProcessGroups {
+		if processGroup.Remove && removedProcessGroups[processGroup.ProcessGroupID] {
+			addresses = append(addresses, processGroup.Addresses...)
+			hasStatusUpdate = true
+		} else {
+			processGroups = append(processGroups, processGroup)
+		}
+	}
+
+	if len(addresses) > 0 {
+		r.Recorder.Event(cluster, "Normal", "IncludingInstances", fmt.Sprintf("Including removed processes: %v", addresses))
+	}
+
+	err = adminClient.IncludeInstances(addresses)
```

Comment:
Should we move the include call into the if statement above? I assume that the `adminClient.IncludeInstances` just will do nothing but it would be cleaner.